dev/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+  <head>
+    <script src="../src/index.js"></script>
+  </head>
+
+  <body>
+    <div class="container" />
+
+    <script>
+      TeamContributionCalendar.default(".container", ["nkapolcs"])
```

Comment:
```suggestion
      TeamContributionCalendar.default(".container", ["tccemptyuser"])
```

Please add `tccemptyuser` as the default user for development purposes.